extensions/podman/packages/extension/src/extension.spec.ts

```diff
@@ -232,9 +232,9 @@ vi.mock('@podman-desktop/api', async () => {
     },
     env: {
       createTelemetryLogger: vi.fn(),
-      isWindows: (): (() => boolean) => vi.fn(),
-      isMac: (): (() => boolean) => vi.fn(),
-      isLinux: (): (() => boolean) => vi.fn(),
```

Comment:
This was causing all the test to fail 🤔 replaced with default to false solved the testing issue